syn/yosys-uhdm/condarc

```diff
@@ -0,0 +1,2 @@
+channels:
+  - conda-forge
```

Comment:
Add `- defaults` as the first channel. This way the built package won't require `conda-forge` channel during installation.

Reply:
Added